docs/interpreter/pig.md

```diff
@@ -94,4 +99,4 @@ c = group b by Category;
 foreach c generate group as category, COUNT($1) as count;
 ```
 
-Data is shared between `%pig` and `%pig.query`, so that you can do some common work in `%pig`, and do different kinds of query based on the data of `%pig`.
+Data is shared between `%pig` and `%pig.query`, so that you can do some common work in `%pig`, and do different kinds of query based on the data of `%pig`. There's one pig tutorial note in zeppelin for your reference.
```

Comment:
I think it's a bit odd to mention this in doc in the `pig.query` section? perhaps move this to under the parent `Example` section instead? how do we reference tutorial notebook in other documentation?